front-end/src/Register/index.js

```diff
@@ -0,0 +1,108 @@
+import React, { useState, useEffect } from "react";
+import { Button, Col, Container, Row, Form } from "react-bootstrap";
+import { useNavigate } from "react-router-dom";
+
+const Register = () => {
+    const navigate = useNavigate();
+    const [username, setUsername] = useState("");
+    const [password, setPassword] = useState("");
+    const [name, setName] = useState("");
+
+    function sendRegistration() {
+        const reqBody = {
+          username: username,
+          password: password,
+          name: name,
+        };
+    
+        fetch("api/user/register", {
+          headers: {
+            "Content-Type": "application/json",
+          },
+          method: "post",
+          body: JSON.stringify(reqBody),
+        })
+          .then((response) => response.json).then(data => setPassword(""), setUsername(""), setName(""));
+      }

```

Comment:
I have created a "helper" method called "ajax()", it uses "fetch()" under the hood, but it implements some best practices to reduce boilerplate fetch code. See other components that use the "ajax()" method for reference. I think the Login component uses one that would work as a good example